server/routes.js

```diff
@@ -67,6 +67,8 @@ router.get("/_health", (ctx) => (ctx.body = "OK"));
 if (process.env.NODE_ENV === "production") {
   router.get("/static/*", async (ctx) => {
     ctx.set({
+      "Service-Worker-Allowed": "/",
```

Comment:
Allows the service worker to access a scope higher level than where it is hosted under /static